bigquery_etl/shredder/config.py

```diff
@@ -69,6 +70,12 @@ def dataset_id(self):
     table="telemetry_stable.deletion_request_v4",
     field="payload.scalars.parent.deletion_request_impression_id",
 )
+CFR_SRC = DeleteSource(
+    # inject sql via f"`{sql_table_id(source)}`" to select client_id and impression_id
+    table="telemetry_stable.deletion_request_v4`,"
+    f" UNNEST([{CLIENT_ID}, {IMPRESSION_SRC.field}]) AS `_",
+    field="_",
+)
```

Comment:
I did this instead of repeating cfr tables with multiple sources, so that we don't have to scan the tables twice, because repeating a table would probably cause issues attempting to update the same partition simultaneously, especially because jobs are sorted by table and partition.

Reply:
@jklukas hit me up if you don't like this, and we can discuss alternative solutions.

Reply:
This is clever. It's indeed rather surprising, but probably not worth refactoring unless we find other tables that also have this problem.